app/Http/Requests/CommandCenterReservationCreateRequest.php

```diff
@@ -29,7 +29,6 @@ public function rules()
         $oneMonthLater =  Carbon::now()->addMonth();
         $shift = CommandCenterShift::find($this->command_center_shift_id);
```

Comment:
```suggestion
        $shift = CommandCenterShift::find($this->input('command_center_shift_id'));
```
sebaiknya ditambahkan input buat jaga jaga jika variable command_center_shift_id nya tidak dikirim sama sekali akan menyebabkan error query bukan error validation

Reply:
ini saya udah coba tetep jalan kang, tapi untuk jaga jaga lebih baik saya tambahkan input

Reply:
iya case nya itu ketika dari frontend atau dari postman gak mengirimkan `command_center_shift_id`